day-01/day-01-main.cc

```diff
@@ -65,8 +65,9 @@ std::tuple<int, int, int> Get3Sum(int target_sum, const std::vector<int>& int_li
 int main(int argc, char **argv) {
   int target_sum = 2020;
 
-  std::string filepath(
-    "C:/Users/grhousto/advent-of-code/day-01/advent-day-01-data.csv");
+  constexpr char filepath[] =
+    "../day-01/advent-day-01-data.csv";
```

Comment:
Has to be run from the build/ directory (or day*/ I guess). Ideal would be getting the executable current directory and appending this.